### PR TITLE
fix reference test failing locally

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/paas 

silently failed for me because:
 - I have gpg signing active
 - it did not raise when it failed

### CODE REVIEW

File: test/lib/references_service_test.rb [hunk 1]
```diff
@@ -3,28 +3,24 @@
...
-    execute_on_remote_repo(tmp, cmds)
-    tmp
+    create_repo_with_tags
+    execute_on_remote_repo("git checkout -b test_user/test_branch")
   end
 
-  let!(:project) { Project.create!(name: 'test_project', repository_url: repository_url) }
+  let!(:project) { Project.create!(name: 'test_project', repository_url: repo_temp_dir) }
 
   before do
```
Comment (irwaters) [inline test/lib/references_service_test.rb]: where is repo_temp_dir defined?

**Comment by irwaters** [comment]
looks fine other than my question.

👍

File: test/lib/references_service_test.rb (hunk 1 above)
Comment (grosser) [inline test/lib/references_service_test.rb]: comes with GitRepoTestHelper

**Comment by irwaters** [comment]
thought as much, thanks.

**Pull request merged**